src/atoms/BlockLink.js

```diff
@@ -1,9 +1,12 @@
 import styled from 'styled-components';
 
 const BlockLink = styled.a`
+	position: relative;
 	display: block;
 	color: ${props => props.theme.colors.textColor};
 	text-decoration: none;
+	overflow: hidden;
```

Comment:
Do you really need overflow hidden on all block-level links, or just some of them?